doc/rpc-spec.yaml

```diff
@@ -0,0 +1,1383 @@
+openapi: 3.0.1
+info:
+  title: Pocket Network
+  description: This is the API definition Pocket Network core RPC calls. Pocket is a distributed network that relays data requests and responses to and from any blockchain system. Pocket verifies all relayed data and proportionally rewards the participating nodes with native cryptographic tokens.
+  termsOfService: 'https://pokt.network/terms/'
+  contact:
+    email: hola@pokt.network
+  license:
+    name: MIT License
+    url: 'https://github.com/pokt-network/pocket-core/LICENSE.md'
+  version: 0.0.1
```

Comment:
Should we increment this to 0.1.0?  or 0.2.0 if that is when it will be published?

Reply:
I Agree!

Reply:
Changing it to 0.2.0